rook/README.md

```diff
@@ -51,6 +51,19 @@ $ popper run -f workflows/iperf.yml teardown
 $ popper run -f workflows/iperf.yml plot-results
 ```
 
+## fio benchmarks
+
+### Installing `kubestone`
```

Comment:
Since the iperf benchmark is also assuming fio, we can add a `kubestone benchmarks` section, and then put the fio and iperf as subsections of those

Reply:
Ah. Thats better.